### ISSUE TITLE
Unity Ads SDK fail to initialize due to game with ID ******* is not enabled

### ISSUE THREAD

**Issue by vhnunes** [AppLovin/AppLovin-MAX-Unity-Plugin#196]
### MAX Plugin Version

5.4.7

### Unity Version

2020.3.37

### Device/Platform Info

Android Devices

### Current Behavior

After upgrading the max plugin to 5.4.7 the Unity Ads adapter is returning (-5001) and is showing errors.

```
Error UnityAds com.unity3d.services.core.configuration.InitializeThread$InitializeStateCreate.execute() (line:23) :: Unity Ads SDK fail to initialize due to game with ID ******* is not enabled

Error UnityAds com.unity3d.services.core.api.Sdk.logError() (line:1) :: Initialization error: Unity Ads SDK fail to initialize due to game with ID ******* is not enabled

Error UnityAds com.unity3d.services.core.configuration.InitializeThread$InitializeStateError.execute() (line:1) :: Unity Ads init: halting init in create_webview_game_id_disabled: Unity Ads SDK fail to initialize due to game with ID ******* is not enabled

Error UnityAds com.unity3d.services.core.configuration.InitializationNotificationCenter.triggerOnSdkInitializationFailed() (line:3) :: SDK Failed to Initialize due to Unity Ads failed to initialize due to internal error
```

### Expected Behavior

No error.

### How to Reproduce

1. Update Unity Max to 5.4.7
2. Add Unity Mediation

### Additional Info

Already checked if the error informed Game ID is matching the Unity Ads portal Android Game ID.
Android SDK Tested was 32 and 33. 32 was working before on previous version of max plugin sdk.
Unity Adapter 4.3.0.0.

**Comment by Lorenzo45** [comment]
Hi @vhnunes:
1. Which MAX Unity Plugin version were you using previously?
2. Did you update the Unity adapter as well? If so, which version were you on before?
3. Have you tried reaching out to Unity Ads support about this issue?
4. If you haven't found a solution yet, can you send us verbose logs from an app session where you get this error? Instructions to enable verbose logs in the AppLovin SDK can be found here: https://support.applovin.com/hc/en-us/articles/4413218413325-How-to-enable-verbose-logging-in-the-AppLovin-SDK

**Comment by vhnunes** [comment]
Hey @Lorenzo45,

About your questions:
1. Previous working MaxPlugin was **5.4.5**
2. Was a clean install of **MaxPlugin (5.4.7)** + unity adapter with latest available versions **(4.3.0.0).** The version of unit ads adapter that was working before is the same that is not working now, **4.3.0.0**
3. Not yet.

**Comment by VanshMillion** [comment]
You solved the issue?
